src/Moryx.Orders/Advice/IAdviceExecutor.cs

```diff
@@ -8,13 +8,8 @@ namespace Moryx.Orders.Advice
     /// <summary>
     /// API for an advice executor. The executor applies an advice of pick parts or orders to an sub system
     /// </summary>
-    public interface IAdviceExecutor : IConfiguredPlugin<AdviceExecutorConfig>
+    public interface IAdviceExecutor : IAsyncConfiguredPlugin<AdviceExecutorConfig>
     {
-        /// <summary>
-        /// Indicator if the advice need user action or can be done automatically
-        /// </summary>
-        bool RequiresUserAction { get; }
```

Comment:
Can/should we just remove this indicator? We have a similar concept for setups, indicating whether they are automatic or require manual work

Reply:
We can, it is not used in our APIs. If we should? Yes because it is unused and when we need some kind of stuff, we can implement it later.